Homework Submission from Alex Grover

apgrover opened the pull request.

exploriamaker committed:
Built main HTML

exploriamaker committed:
Core CSS in place

exploriamaker committed:
Added images, finalized CSS style

- added fonts
- added images
- added final CSS

peter-avila commented on css/styles.css:
```diff
@@ -0,0 +1,73 @@
+  padding: 1em;
+  text-align: center;
+}
+.header__nav {
```
Great use of BEM.  👍

peter-avila commented on css/styles.css:
```diff
@@ -0,0 +1,73 @@
+  font-weight: bold;
+}
+.nav__links {
+  margin: 0 .5em;
```
+1 for using non-pixel units.  👍

peter-avila commented on css/styles.css:
```diff
@@ -0,0 +1,73 @@
+.nav__links__current {
+  margin: 0 .5em;
+  color: #FFFFFF;
+  font-family: 'Signika', sans-serif;
```
Good use of a fallback font.  👍  Consider setting broad default styles on the `body` tag, then overwrite as necessary.

peter-avila commented on css/styles.css:
```diff
@@ -0,0 +1,73 @@
+  font-family: 'Signika', sans-serif;
+}
+.main {
+  margin: 0 5%;
```
You should consider applying padding/margin with em, rem, or even px. Using percentages may yield unexpected results at some window dimensions.

peter-avila commented on css/styles.css:
```diff
@@ -0,0 +1,73 @@
+}
+.main {
+  margin: 0 5%;
+  width: auto;
```
This rule may be unnecessary.  🔨

peter-avila commented on css/styles.css:
```diff
@@ -0,0 +1,73 @@
+  font-size: .7em;
+  font-family: 'Signika', sans-serif;
+}
+.full-width {
```
Nice use of a class as a general modifier.  👍 This will come in handy when you make your own grid layouts.

peter-avila commented on experience.html:
```diff
@@ -0,0 +1,67 @@
+<head>
+  <title>Godzilla for Hire | Experience/Interests</title>
+  <link rel="stylesheet" type="text/css" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Bowlby+One+SC|Signika" rel="stylesheet">
```
Like the use of Google Fonts. 👍

peter-avila commented on experience.html:
```diff
@@ -0,0 +1,67 @@
+  <header class="header">
+    <nav class="header__nav">
+      <a class="nav__links" href="index.html">Bio</a> | 
+      <a class="nav__links__current" href="#">Experience/Interests</a>
```
this should probably read: 

`<a class="nav__links nav__links--current" href="#">Experience/Interests</a>`

🔨

peter-avila commented on experience.html:
```diff
@@ -0,0 +1,67 @@
+  <main class="main">
+    <h1 class="main__h1">Experience/Interests</h1>
+    <h2 class="main__h2">Cities Attacked/Destroyed</h2>
+    <ul class="main__unordered">
```
Killing it with pristine structure. 👍

peter-avila commented on experience.html:
```diff
@@ -0,0 +1,67 @@
+      <li class="main__listitem">Forum on Innovation and Feasibility of Destroying Tokyo (FIFDT)</li>
+    </ul>
+  <div class="full-width">
+    <img class="wide-image" src="assets/godzilla_3.jpg" alt="" />
```
Although not required, the best practice is to include the `alt` attribute which allows for accessibility to the visually impaired and is what appears if the resource fails to load. 🔨

peter-avila commented on index.html:
```diff
@@ -0,0 +1,42 @@
+      Godzilla is sure to be a great asset on any team bent on destroying the world.
+    </p>
+    <p class="main__copy">
+      To see what Godzilla is up to nowadays, read <a href="https://www.nytimes.com/2014/05/16/movies/godzilla-grandaddy-of-movie-monsters-stomps-back.html?_r=0" alt="Remember, Godzilla is watching you. Probably should click the link... for all our sakes">this review</a> by the <em>New York Times</em> to see what the green, towering, fearsome hype is all about!
```
The common convention for external links is to open them up in a new tab/window.  🔨

peter-avila reviewed (changes requested):
Really great job overall.  Your code is very tidy and you used many of today's best practices for web dev.  

Please take a look at the comment(s) and make any needed corrections.

exploriamaker committed:
Revisions for Homework #1

- added target=“_blank” to link
- removed .main css
- cleaned up/condensed css
- added font family stack to body
- fixed styles on navigational links
- added alt text
